cbor-schema/plt.cddl

```diff
@@ -194,3 +188,24 @@ token-module-state = {
     ; of which are not defined in the present specification.
     * text => any
 }
+
+; Token module events.
+
+; These events are limited to the events emitted by the token module. Events related to protocol-level tokens
+; balance changes are defined from the token kernel.
```

Comment:
I suggest de-emphasising the token module/token kernel a bit, and rather say that token balance change events are defined separately and not subject to CBOR encoding.